From 4cbf32be0ac85dbcd09b196ae3951755bf9c46b9 Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Sun, 14 Apr 2013 15:24:42 +0200
Subject: [PATCH 37/65] r92su: fix sizeof(pointer) bug

Discovered by: coccinelle
r92su/tx.c:290:20-26: ERROR: application of sizeof to pointer

Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
---
 r92su/tx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/wireless/r92su/tx.c
+++ b/drivers/net/wireless/r92su/tx.c
@@ -287,7 +287,7 @@ r92su_tx_prepare_tx_info_and_find_sta(st
 	}
 
 	/* clean up tx info */
-	memset(tx_info, 0, sizeof(tx_info));
+	memset(tx_info, 0, sizeof(*tx_info));
 
 	sta = r92su_sta_get(r92su, hdr->h_dest);
 	if (!sta) {
